docker-compose-dev.yml

```diff
@@ -19,22 +19,25 @@ services:
     ports:
       - "9000:9000"
 
-  xain-fl:
+  xain-fl-dev:
     build:
       context: .
       dockerfile: Dockerfile.dev
     command: coordinator -f test_array.npy
     volumes:
-      - '.:/app'
       #  don't use the local egg-info, if one exists
-      - '/app/xain_fl.egg-info'
+      - /app/xain_fl.egg-info
```

Comment:
What does it do when there's no `:` ? Docker mounts the local `xain_fl.egg-info` if it exists and does nothing otherwise?

Reply:
The engine handles the mount, this is only to explicitly don't use the local `egg-info`, even if you mount the entire directory